AFQ/afq_bids_entities.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "bidsafq",
+    "entities": [
+      {
+        "name": "odfmodel",
+        "pattern": "[_/\\\\]+odfmodel-([a-zA-Z0-9]+)"
+      },
+      {
+        "name": "coordsys",
+        "pattern": "[_/\\\\]+coordsys-([a-zA-Z0-9]+)"
+      },
+      {
+        "name": "trkmodel",
+        "pattern": "[_/\\\\]+trkmodel-([a-zA-Z0-9]+)"
+      },
+      {
+        "name": "bundlerec",
```

Along the same lines, maybe this should be:

```suggestion
        "name": "recogmethod",
```

That's so long, though.